app/models/test_track/misconfiguration_notifier.rb

```diff
@@ -1,13 +1,32 @@
-class TestTrack::MisconfigurationNotifier
-  def notify(msg)
-    raise msg if Rails.env.development?
+module TestTrack::MisconfigurationNotifier
+  class Wrapper
+    attr_reader :underlying
 
-    Rails.logger.error(msg)
+    def initialize(underlying = Null.new)
+      @underlying = underlying
+    end
+
+    def notify(msg)
+      raise msg if Rails.env.development?
+
+      Rails.logger.error(msg)
+
+      underlying.notify(msg)
+    end
+  end
+
+  class Null
+    def notify(_); end
+  end
 
-    if Airbrake.respond_to?(:notify_or_ignore)
-      Airbrake.notify_or_ignore(StandardError.new, error_message: msg)
-    else
-      Airbrake.notify(StandardError.new, error_message: msg)
+  class Airbrake
+    def notify(msg)
+      raise "Aibrake was configured not found" unless defined?(::Airbrake)
```

Comment:
`Aibrake was not found`? or is this just missing a comma?

Reply:
oh, i think i need a `but`